src/Plugin/GraphQL/Schema/SdlSchemaPluginBase.php

```diff
@@ -225,7 +225,14 @@ protected function getSchemaDefinition() {
     $file = "{$module->getPath()}/graphql/{$id}.graphqls";
 
     if (!file_exists($file)) {
-      throw new InvalidPluginDefinitionException(sprintf("Missing schema definition file at %s.", $file));
+      throw new InvalidPluginDefinitionException(
+        $id,
+        t('The module "@name" needs to have a schema definition "@file" in its folder for "@plugin" to be valid.', [
```

Comment:
t() is forbidden in exceptions, see https://www.drupal.org/docs/develop/coding-standards/php-exceptions . Please use sprintf() or concatenate the strings.

Reply:
Replaced with strtr().